src/async_fuse/memfs/kv_engine/etcd_impl.rs

```diff
@@ -207,11 +434,117 @@ impl KVEngine for EtcdKVEngine {
         }
     }
 
+    /// Range get, return all key-value pairs start with prefix
     async fn range(&self, prefix: &KeyType) -> DatenLordResult<Vec<ValueType>> {
         let _timer = KV_METRICS.start_kv_operation_timer("range");
         let result = self.range_raw_key(prefix.to_string_key()).await?;
         Ok(result)
     }
+
+    /// Revokes a lease
+    async fn lease_revoke(&self, lease_id: i64) -> DatenLordResult<()> {
+        let mut client = self.client.clone();
+        client
+            .lease_revoke(lease_id)
+            .await
+            .with_context(|| "failed to revoke lease".to_owned())?;
+        Ok(())
+    }
+
+    /// Watch the key, return a receiver to receive the value
+    #[allow(clippy::pattern_type_mismatch)] // Raised by `tokio::select`
+    async fn watch(
+        &self,
+        prefix: &KeyType,
+    ) -> DatenLordResult<Arc<mpsc::Receiver<(String, Option<ValueType>)>>> {
```

Comment:
Remove the Arc on mpsc::Receiver,   since mpsc dose not allow multi receiver.

Reply:
Ok, that make sense, I have removed it.